jwala-persistence/src/test/java/com/cerner/jwala/persistence/service/webserver/impl/JpaWebServerPersistenceServiceImplTest.java

```diff
@@ -47,7 +41,15 @@ public WebServerPersistenceService getWebServerPersistenceService(){
 
         @Bean
         public GroupPersistenceService getGroupPersistenceService(){
-            return new JpaGroupPersistenceServiceImpl(getGroupCrudServiceImpl(), getGroupJvmRelationshipService());
+            return new JpaGroupPersistenceServiceImpl(
```

Comment:
Minor: I know you didn't format it this way since you just added a parameter but it would look nicer if it were formatted like this:

```java
return new JpaGroupPersistenceServiceImpl(getGroupCrudService(), getGroupJvmRelationshipService(), 
                                          getApplicationCrudService());
```